src/common/schema/application.js

```diff
@@ -128,9 +128,19 @@ module.exports = {
       minimum: 0,
       maximum: 9999999.99,
     },
+    states: {
+      oneOf: [
+        {
+          'enum': states
+        }, {
+          type: 'string',
+          pattern: '^$'
+        }
+      ]
+    },
     phone: {
       type: 'string',
-      pattern: '^[0-9]{10}$'
+      pattern: '^[0-9]{10}$|^$'
```

Comment:
Do you want to the phone number type itself is always allowed to be blank, or do we care more about specific uses of it (i.e. homePhone must exist, but spousePhone can be blank)?